app/models/campaign.rb

```diff
@@ -0,0 +1,12 @@
+class Campaign < ApplicationRecord
+  belongs_to :user
+  validates :title, presence: true,
+                    uniqueness: true,
+                    length: { minimum: 3 }
+  validates :description, presence: true
+  validates :current_amount, :needed_amount, presence: true,
+                                             numericality: { greater_than_or_equal_to: 0 }
+  validates :requisite, presence: true,
+                        numericality: true,
```

Comment:
will it be valid, if I will enter "+11122223333444555" or "-111222233334444"?

Reply:
No

Reply:
Yes,
I will add < format: { with: /\A\d+\z/ } > for validation